Pipfile

```diff
@@ -32,6 +32,9 @@ python-dateutil = "*"
 bs4 = "*"
 azure = "==4.0.0"
 azure-mgmt-costmanagement = "==0.1.0"
+isodate = "==0.6.0"
+azure-storage-blob = "*"
+azure-storage = "*"
```

Comment:
How do you feel about unlocked versus version-locked dependencies?

I could go either way, but seeing as most everything else is version-locked, we may just want all new deps to be locked as well.

Reply:
If we lock, pyup should just keep advancing the version number anyway, so they're effectively equal.

Reply:
@adberglund  - The difference, I think, is that pyup won't do major-version upgrades. It only does incremental upgrades where `*` will accept any version, including a major update.

Reply:
I meant to come back later and lock these.  Thanks for the reminder!